molgenis-platform-integration-tests/src/test/java/org/molgenis/integrationtest/platform/PlatformITConfig.java

```diff
@@ -81,9 +80,9 @@
 		org.molgenis.data.RepositoryCollectionBootstrapper.class, org.molgenis.data.EntityFactoryRegistrar.class,
 		org.molgenis.data.importer.emx.EmxImportService.class, org.molgenis.data.importer.ImportServiceFactory.class,
 		org.molgenis.data.FileRepositoryCollectionFactory.class, org.molgenis.data.excel.ExcelDataConfig.class,
-		org.molgenis.security.permission.PermissionSystemService.class,
```

Comment:
The Import annotation value requires classes annotated with Configuration or regular component classes. Due to changes in the pull request PermissionSystemService is an interface and PermissionSystemServiceImpl is the component.

I suspect the Import annotation is used here to include specific components instead of including all components in its package which could result in dependency resolution issues.